pkg/autoid_service/autoid.go

```diff
@@ -149,7 +158,11 @@ func (alloc *autoIDValue) alloc4Signed(ctx context.Context,
 				return err1
 			}
 			// calcNeededBatchSize calculates the total batch size needed on global base.
-			n1 = calcNeededBatchSize(newBase, int64(n), increment, offset, isUnsigned)
+			if alloc.base == 0 {
```

Comment:
When it will equals to `0`?

Reply:
When the service is not initialized ... it need to get the value from remote.